tutorials/Native_distributions_and_local_execution/README.md

```diff
@@ -637,3 +637,14 @@ compose.desktop {
     }
 }
 ```
+
+ProGuard's optimization is disabled by default since Compose 1.5. To enable it, set the following property via Gradle DSL:
+```
+compose.desktop {
+    application {
+        buildTypes.release.proguard {
+            optimize.set(true)
```

Comment:
Tested on Compose 1.4.1, Windows

[benchmark](https://github.com/JetBrains/compose-multiplatform/tree/igor.demin/benchmark-median1)

`./gradlew run`, OpenJDK 20
```
AnimatedVisibility: 26ms
LazyGrid: 101ms
VisualEffects: 33ms
```
`./gradlew run`, OpenJDK 18
```
AnimatedVisibility: 27ms
LazyGrid: 104ms
VisualEffects: 33ms
```

`./gradlew runDistributable`, OpenJDK 18:
```
AnimatedVisibility: 28ms
LazyGrid: 102ms
VisualEffects: 33ms
```

`./gradlew runReleaseDistributable`, OpenJDK 18:
```
AnimatedVisibility: 27ms
LazyGrid: 98ms
VisualEffects: 33ms
```

[benchmark](https://github.com/JetBrains/compose-multiplatform/tree/igor.demin/benchmark-median1-startup), startup + single frame, LazyGrid, median, 10-15 tries:
`./gradlew run`, OpenJDK 20
```
311ms
```
`./gradlew run`, OpenJDK 18
```
304ms
```

`./gradlew runDistributable`, OpenJDK 18:
```
306ms
```

`./gradlew runReleaseDistributable`, OpenJDK 18:
```
261ms
```

When we apply optimizations - no significant improvements when we render frames, but there are some improvement in the startup time.

So, it is preferable to keep optimizations working - [this PR](https://github.com/JetBrains/compose-multiplatform-core/pull/706) fixes this particular issue. Also I created [a task](https://github.com/JetBrains/compose-multiplatform/issues/3404) for the future - run all Compose tests after ProGuard optimizations.

As for disabling optimizations by default - it is debatable, let's just keep them enabled until there will be a strong argument against it.

Reply:
Ok, let's enable optmizations again but keep it in Gradle DSL